Godeps/Godeps.json

```diff
@@ -324,6 +326,55 @@
 			"Comment": "v1.12.7",
 			"Rev": "760741802ad40f49ae9fc4a69ef6706d2527d62e"
 		},
+		{
+			"ImportPath": "github.com/bazelbuild/buildifier/core",
+			"Rev": "f348e34d6a4005498ed8875df39762356a8b0efa"
+		},
+		{
+			"ImportPath": "github.com/bazelbuild/buildtools/build",
+			"Comment": "0.6.0-60-g1a9c38e",
+			"Rev": "1a9c38e0df9397d033a1ca535596de5a7c1cf18f"
+		},
+		{
+			"ImportPath": "github.com/bazelbuild/buildtools/tables",
+			"Comment": "0.6.0-60-g1a9c38e",
+			"Rev": "1a9c38e0df9397d033a1ca535596de5a7c1cf18f"
+		},
+		{
+			"ImportPath": "github.com/bazelbuild/rules_go/go/tools/gazelle/config",
+			"Comment": "0.8.1-1-g737df20",
+			"Rev": "737df20c53499fd84b67f04c6ca9ccdee2e77089"
+		},
+		{
+			"ImportPath": "github.com/bazelbuild/rules_go/go/tools/gazelle/gazelle",
```

Comment:
note for when you revisit this: the 0.8.0 tag in `bazelbuild/bazel-gazelle` is compatible with the version of gazelle we're using now, and is preferred over the gazelle in `rules_go`.

Reply:
no `0.8.0`, but `0.8` matches the hash.